.gitignore

```diff
@@ -0,0 +1,40 @@
+.deps
+.libs
+Makefile
+Makefile.fragments
+Makefile.global
+Makefile.objects
+acinclude.m4
+aclocal.m4
+autom4te.cache
+build
+config.guess
+config.h
+config.h.in
+config.log
+config.nice
+config.status
+config.sub
+configure
+configure.in
+configure.ac
+install-sh
+libtool
+ltmain.sh
+missing
+mkinstalldirs
+modules
+run-tests.php
+
+*.la
+*.lo
+*.o
+
+*.swp
+.*.rej
+*.rej
+.*~
+*~
+.#*
+.DS_Store
+Thumbs.db
```

Comment:
These last two should really be in your global .gitignore.

Reply:
Removed `.DS_Store` and `Thumbs.db` from .gitignore.